microprofile/microprofile-rest-client-1.3/src/main/resources/META-INF/microprofile-config.properties

```diff
@@ -0,0 +1,6 @@
+# This properties override any baseUri in annotation (e.g. @RegisterRestClient(baseUri = "http://localhost:1234"))
+# baseUri on ClientCloseable is overridden by myClients/mp-rest/url
+org.wildfly.swarm.ts.microprofile.rest.client.v13.ClientCloseable/mp-rest/url=http://localhost:666
```

Comment:
Will these work in an unprivileged workspace? IMHO we should avoid the ports under 1023 to make sure everything works in all environments.

Reply:
It seems the application will never listen on 666 (or 777), so not sure what this configuration means. Is it supposed to provoke an error @penehyba?

Reply:
myClients/mp-rest/uri (mind the "i") was supposed to override all nonsense configurations before...anyway, I removed this as this is unlikely to occur.

Reply:
Ha, I didn't know that. Could you add it back? Just use `http://nonsense` instead of `localhost:666` or something like that :-)